src/pvrclient-mediaportal.cpp

```diff
@@ -2481,3 +2482,52 @@ PVR_ERROR cPVRClientMediaPortal::GetStreamReadChunkSize(int& chunksize)
   chunksize = 32 * 1024;
   return PVR_ERROR_NO_ERROR;
 }
+PVR_ERROR cPVRClientMediaPortal::GetRecordingEdl(const kodi::addon::PVRRecording& recording, std::vector<kodi::addon::PVREDLEntry>& edl) // bilkusg
+// liberally adapted from pvr.vuplus ( recordings.cpp in particular)
+{
+    cRecording* myrecording = GetRecordingInfo(recording);
+    const char* pvrFilePath = myrecording->FilePath();
+    const std::string& pvrSmbPath = m_tsreader->TranslatePath(pvrFilePath);
+    const std::string edlFile = pvrSmbPath.substr(0, pvrSmbPath.find_last_of('.')) + ".edl";
```

Comment:
Need to guard for a nullptr coming back from GetRecordingInfo.

```cpp
    cRecording* myrecording = GetRecordingInfo(recording);
    if (!cRecording)
      return PVR_ERROR_SERVER_ERROR;

    const std::string pvrSmbPath = m_tsreader->TranslatePath(myrecording->FilePath());
    const std::string edlFile = pvrSmbPath.substr(0, pvrSmbPath.find_last_of('.')) + ".edl";
```

Reply:
OK